galsim/phase_psf.py

```diff
@@ -1340,7 +1348,8 @@ class OpticalPSF(GSObject):
 
     def __init__(self, lam_over_diam=None, lam=None, diam=None, tip=0., tilt=0., defocus=0.,
                  astig1=0., astig2=0., coma1=0., coma2=0., trefoil1=0., trefoil2=0., spher=0.,
-                 aberrations=None, aper=None, circular_pupil=True, obscuration=0., interpolant=None,
+                 aberrations=None, annular_zernike=False,
```

Comment:
Added `annular_zernike` to `_opt_params`.  Just talked to Mike about `aberrations`.  It turns out these are handled by a custom builder object in `config/gsobject.py`, so they are (still) supported.